test/UnitTests/HomeControllerTest.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Data.Entity;
+using Microsoft.Framework.Cache.Memory;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+using Xunit;
+using MusicStore.Controllers;
+using MusicStore.Models;
+using System.Linq;
+
+namespace UnitTests
+{
+    public class HomeControllerTest
+    {
+        IServiceProvider _serviceProvider;
+
+        public HomeControllerTest()
+        {
+            var services = new ServiceCollection();
+
+            services.AddEntityFramework()
+                      .AddInMemoryStore()
+                      .AddDbContext<MusicStoreContext>();
+
+            _serviceProvider = services.BuildServiceProvider();
+        }
+
+        [Fact]
+        public void Error_ReturnsErrorView()
+        {
+            // Arrange
+            var controller = new HomeController();
+            var errorView = "~/Views/Shared/Error.cshtml";
+
+            // Act
+            var result = controller.Error();
+
+            // Assert
+            var viewResult = Assert.IsType<ViewResult>(result);
+
+            Assert.Equal(errorView, viewResult.ViewName);
```

Comment:
Thanks. I'll make sure to remove this test when we remove the corresponding action.